wix.json

```diff
@@ -0,0 +1,28 @@
+{
+  "product": "Contrail docker driver",
+  "company": "Juniper",
+  "license": "LICENSE.rtf",
+  "upgrade-code": "ca674931-876d-4dfb-bb67-136c585b6ffc",
+  "files": {
+    "guid": "ca8cbb44-2a0d-4f95-9375-dccabc92ffc1",
+    "items": [
+      "../../../../bin/contrail-windows-docker.exe"
```

Comment:
Could you use here some variable for "../../"? For example "[INSTALLDIR]\\contrail-windows-docker.exe\ " ?